src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1962,7 +1962,7 @@ def get_crash_data(crash_data, symbolize_flag=True):
       continue
 
   # Only get repo.prop if we have an Android kernel or KASAN crash
-  if not state.android_kernel_repo and (found_android_kernel_crash or is_kasan):
+  if environment.is_android() and (found_android_kernel_crash or is_kasan):
```

Comment:
Just fyi @mspectorgoogle, this would need a rebase in your cl, since this block runs outside loop, there is no need to check state.android_kernel_repo. in your cl, you can even remove this var from constructor and then just use a local var inside this block.